x-pack/plugins/security_solution/public/detection_engine/rule_management/components/rule_details/rule_details_flyout.tsx

```diff
@@ -196,7 +196,7 @@ export const RuleDetailsFlyout = ({
       paddingSize="l"
       data-test-subj={dataTestSubj}
       aria-labelledby={prebuiltRulesFlyoutTitleId}
-      ownFocus
```

Comment:
I am not sure that this is the correct change, we should not do it just to make the tests pass. This table is partially (or completely) covered by the open `Flyout`, which means, from an `a11y` perspective, we should not allow setting focus on it.

Let's ask EUI. @cee-chen / @1Copenut  please give us your input.

Reply:
I see the importance of the functionality for quick preview, but also I think we need to follow a consistent style for when it is appropriate to use `ownFocus` and when it is not.

Reply:
@nikitaindik I have the same concern. It's better to follow current UX across Kibana. I'm pretty sure the test you mentioned was written some time ago when the flyout didn't own focus by default.

Reply:
EuiFlyout does [set `ownFocus={true}` by default](https://github.com/elastic/eui/blob/5c4031565d1f0fef7e08e2377ea5b29bf8963b54/packages/eui/src/components/flyout/flyout.tsx#L182), so unless this component now explicitly sets `ownFocus={false}` this component should be behaving the same way as before.

Reply:
Ah sorry disregard my previous unhelpful comment, I was looking only at the line highlighted and not at the full diff, I see this is setting `ownFocus={false}`.

`ownFocus={false}` flyouts still move/trap focus, so the flyout content still remains accessible for keyboard users, although the content below it does not: https://eui.elastic.co/#/layout/flyout#without-ownfocus

Alexey, does the above documentation help explain `ownFocus` usage?

Reply:
@cee-chen Yep, that's exactly what I mean. That case doesn't seem like something where we should set `ownFocus={false}`. The default behavior will provide the best UX

Reply:
Thanks for your input folks! I have tried both options myself and found that when tabbing using the keyboard the focus seems to behave the same. With either `ownFocus={false}` or `ownFocus={true}` it stays within the flyout and top page menu. Please tell me if I misunderstood your concern.

Also I have discussed this with our PO (@approksiu) and design team today, showed them both options - with and without the backdrop. And we have concluded that for our use case (quickly switching between rules) it would be desirable if there's no backdrop and table items are still clickable with a mouse.

Reply:
I certainly support the unification of UX, but this is your decision. Please keep in mind that this will lead to the following issues:

1. When switching between table elements, the focus will not move to the EuiFlyout as expected.
2. There may be incorrect behaviour of the main content scrolling when using keyboard navigation inside the EuiFlyout. See same problem we have here https://github.com/elastic/kibana/pull/192372 . I've tested for your case and issue is here.

I'm not insisting, I just want to highlight and explain why we removed `ownFocus={false} `

@approksiu just fyi

Reply:
Thanks for highlighting these 2 issues @alexwizp! I have discussed these with the team and we've decided to revert to the default behaviour (with the dark backdrop). Made the change in this commit: https://github.com/elastic/kibana/pull/191978/commits/d1cadfa37cf314da0f6ffcb24f4c7e584e1851fb